dbt-requirements.txt

```diff
@@ -1,2 +0,0 @@
-dbt-postgres==1.3.1
-pytz
```

Comment:
As mentioned before, I suggest we restore this file (happy to have `dbt-postgres~=1.7.4`)